js/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@mosip/svg-renderer",
+  "version": "0.1.0",
+  "repository": "https://github.com/mosip/inji-vc-renderer",
+  "author": "MOSIP",
+  "license": "MIT",
+  "description": "JS library to render VC with SVG template support",
+  "main": "VCRenderer.js",
+  "scripts": {
+    "example": "cd example && node example-app.js && cd ..",
+    "test": "jest"
```

Comment:
Add verify script to run test
`"verify": "npm test"`

Reply:
Done